test/acceptance/features/steps/openshift.py

```diff
@@ -9,17 +9,19 @@
 class Openshift(object):
     def __init__(self):
         self.cmd = Command()
-        self.operator_source_yaml_template = '''
----
-apiVersion: operators.coreos.com/v1
-kind: OperatorSource
+        self.catalog_source_yaml_template = '''
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
 metadata:
     name: {name}
     namespace: openshift-marketplace
 spec:
-    type: appregistry
-    endpoint: https://quay.io/cnr
-    registryNamespace: {registry_namespace}
+    sourceType: grpc
+    image: {registry_image}
```

Comment:
Just curious, Now that operator source is deprecated, this registry is not the quay app registry ? It can be simply an image of the bundle?

Reply:
Actually it is a "operator catalog" rather then "app registry". According to the [official docs](https://docs.openshift.com/container-platform/4.4/release_notes/ocp-4-4-release-notes.html#ocp-4-4-marketplace-apis-deprecated) we need to "build the operator catalog image from the quay app registry". So, I see that the name of the variable `{registry_image}` is a bit misleading - so I'll rename it to the `catalog_image`.

It can't be simply an image of the bundle, it is supposed to be an operator catalog to be consumed by the openshift marketplace